src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck starts work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stopped working");
```

Comment:
Similarly, the message 'Truck stopped working' could provide more context about the state of the Truck when it stops working. More informative messages would improve the clarity of the code behavior.